src/transport/SecureSession.h

```diff
@@ -60,10 +62,10 @@ class DLL_EXPORT SecureSession
      *   Derive a shared key. The derived key will be used for encryting/decrypting
      *   data exchanged on the secure channel.
      *
-     * @param remote_public_key  A pointer to peer's public key
-     * @param public_key_length  Length of remote_public_key
-     * @param local_private_key  A pointer to local private key
-     * @param private_key_length Length of local_private_key
+     * @param secret             A pointer to the share secret
```

Comment:
this should be "the shared secret" right?

Reply:
Thanks ! Fixed.